### ISSUE TITLE
Random Lag

### ISSUE THREAD

**Issue by talon19912** [arenasys/qDiffusion#90]
**Describe the bug**
I keep running into an issue where the program lags for no apparent reason. When i connect to colab, its random whether the program work as it should. Under normal circumstances it should respond instantly, but for some reason it has a massive input lag. It takes somewhere between 30 seconds to up to 2 minutes to switch between tabs in the ui, when i click in the prompts windows to type there's a 30 second to a minute delay before it lets me type, but once i am able to type it works fine, but if i click any where else to do something, like another window and click back on the ui it lags again for up to a minute before it responds. Now I'm pretty sure its not my hardware because some times the program will run perfectly and i'll have no issues at all.

Lately I've been getting a weird encoder/decoder bug as well when i've never seen it before.

**Comment by mmsspgetti** [comment]
Yes, I also encountered the second problem, but this happens at the UNET download stage, and in this case I can no longer generate anything. This error appears once and then with each generation attempt I get "unknown". Reinstalling, restarting, reconnecting, etc. does not help.
![image](https://github.com/user-attachments/assets/7d5d1dea-4fb2-42e4-ad98-e0ff05f6f6ba)
![image](https://github.com/user-attachments/assets/23fd9c07-004e-4fc2-b107-199a3e9291d8)
